web/src/features/hfiCalculator/slices/hfiCalculatorSlice.ts

```diff
@@ -156,17 +168,27 @@ const dailiesSlice = createSlice({
       state.result = action.payload
       state.dateRange = action.payload?.date_range
       state.loading = false
+    },
+    setFuelTypes: (
+      state: HFICalculatorState,
+      action: PayloadAction<FuelTypesResponse>
+    ) => {
+      state.fuelTypes = action.payload.fuel_types
+      state.loading = false
```

Comment:
Fuel types would then set load to false even though stations could still be loading.

Reply:
Also this should be covered by tests.